testing/endtoend/deps.bzl

```diff
@@ -1,5 +1,8 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")  # gazelle:keep
 
+lighthouse_version = "v2.1.1"
```

Comment:
Would it be possible to fetch the latest version automatically?

Reply:
For e2e we want consistency, dynamically changing the version depending on when you run the command doesn't seem like the best idea. Also if lighthouse updates their version and it breaks the e2e run, we will be stuck with a broken build.